test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetRestoreCommandTest.cs

```diff
@@ -327,6 +327,55 @@ public void RestoreCommand_FromSolutionFile(string configFileName)
             }
         }
 
+        [Theory]
+        [InlineData("packages.config")]
+        [InlineData("packages.proj2.config")]
+        public void RestoreCommand_FromFilteredSolutionFile(string configFileName)
```

Comment:
I don't think we need to check for a different config file name. That's not relevant in the context of a solution filter support test.

Reply:
You're right I used the test above as a base and forgot to remove this.